docs/pages/build-reference/troubleshooting.mdx

```diff
@@ -79,7 +79,7 @@ This can often be a sign that your app bundle is extremely large, which will mak
 
 To determine how large your bundle is and to see a breakdown of where the size comes from, use [react-native-bundle-visualizer](https://github.com/IjzerenHein/react-native-bundle-visualizer).
 
-It's not yet possible to increase memory limits on your build workers, [only one worker configuration is currently available](/build-reference/limitations).
+It's not yet possible to increase memory limits on your EAS Build builders, [only one builder configuration is currently available](/build-reference/limitations).
```

Comment:
I don't think it's true that only one builder configuration is currently available. We have publicly available `large` Android workers. It would be nice if we can get users who hit OOM errors to use `large` workers since they have more RAM.

Reply:
Updated ✅